test/integration/roles/openstack_tk.json

```diff
@@ -2,6 +2,7 @@
   "env_run_lists": {},
   "run_list": [
     "recipe[openstack_test::hosts]",
+    "recipe[openstack_test::cacert]",
```

Comment:
Thanks. This is an interesting setup. Hopefully I can make time to read the code and understand how you are making these things work.